src/index.html

```diff
@@ -9,6 +9,20 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="http://moyo.ua" class="img_container">
+        <img src="images/logo.png" alt="logo_icon">
+      </a>
+      <nav class="brands_container">
+        <a href="https://www.moyo.ua/foto_video/video_tech/" class="brand">ВИДЕО</a>
+        <a href="https://www.moyo.ua/foto_video/photo_video/" class="brand">ФОТО</a>
```

Comment:
```suggestion
        <a href="https://www.moyo.ua/foto_video/photo_video/" class="brand">фото</a>
```
Зазвичай текст пишеться у нижньому регістрі, а у стилях вже використовується text-transform: uppercase